workflow_tests/utils/ExtendedAct.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable @typescript-eslint/dot-notation */
+// This eslint-disable comment is here to allow accessing private properties in the Act class
```

Comment:
I'm not convinced. This is not something we'd like to restrict in this file, because it stems from the workaround we had to apply to access any private properties from the parent class. I'd keep it until it gets resolved on the library level.